### PR TITLE
fix(platform/lambda): remove @prisma/engines (cli binaries) from zip

### PR DESCRIPTION
**Pull request by matthewmueller**
- ~~only use binaryTarget native~~
- remove @prisma/engines from zip

### CODE REVIEW

File: platforms/lambda/prisma/schema.prisma [hunk 1]
```diff
@@ -1,6 +1,6 @@
 generator client {
   provider      = "prisma-client-js"
-  binaryTargets = ["native", "rhel-openssl-1.0.x"]
+  binaryTargets = ["native"]
```
Comment (Jolg42) [inline platforms/lambda/prisma/schema.prisma]: `"rhel-openssl-1.0.x"` is needed for Lambda

See https://www.prisma.io/docs/guides/deployment/deploying-to-aws-lambda#binary-targets-in-schemaprisma

Comment (Jolg42) [reply platforms/lambda/prisma/schema.prisma]: native is to make it works on your local machine

Comment (matthewmueller) [reply platforms/lambda/prisma/schema.prisma]: Why do we end up with `query-engine-debian-openssl-1.1.x` then? I figured lambda' `native` is `query-engine-debian-openssl-1.1.x`

Comment (Jolg42) [reply platforms/lambda/prisma/schema.prisma]: so we end up with 

From .prisma/client (to use for @prisma/client)
```
  adding: node_modules/.prisma/client/query-engine-rhel-openssl-1.0.x (deflated 64%)
  adding: node_modules/.prisma/client/query-engine-debian-openssl-1.1.x 
```

From @prisma/engines (binaries for prisma cli only)
```
  adding: node_modules/@prisma/engines/introspection-engine-debian-openssl-1.1.x (deflated 65%)
  adding: node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x (deflated 64%)
  adding: node_modules/@prisma/engines/prisma-fmt-debian-openssl-1.1.x (deflated 70%)
  adding: node_modules/@prisma/engines/migration-engine-debian-openssl-1.1.x (deflated 63%)
```

Comment (Jolg42) [reply platforms/lambda/prisma/schema.prisma]: Here the CI native is ubuntu which is Debian

Comment (Jolg42) [inline platforms/lambda/prisma/schema.prisma]:
```suggestion
  binaryTargets = ["native", "rhel-openssl-1.0.x"]
```
This is needed so generate works on native (in CI) and on "rhel-openssl-1.0.x" which is lambda

Comment (matthewmueller) [reply platforms/lambda/prisma/schema.prisma]: Ah, makes sense! Fixing!

Comment (Jolg42) [reply platforms/lambda/prisma/schema.prisma]: Just fixed in a comment 😄

Comment (matthewmueller) [reply platforms/lambda/prisma/schema.prisma]: Oh hah, woops. My github workflows are a bit dated 😅

**Pull request merged**